src/components/Views/ManualGPS/ManualGPS.js

```diff
@@ -761,4 +769,4 @@ class ManualGPS extends React.PureComponent<StateProps & DispatchProps, State> {
   }
 }
 
-export default ManualGPS;
+export default withNavigation(ManualGPS);
```

Comment:
Not sure we even need this anymore! The new upgrade of react-navigation auto includes navigation as a prop so we don't need to do withNavigation!

Reply:
ooo fancy, okay!